packages/http-server/src/server.ts

```diff
@@ -90,7 +91,7 @@ export const createServer = (operations: IHttpOperation[], opts: IPrismHttpServe
     pipe(
       TE.fromEither(requestConfig),
       TE.chain(requestConfig => prism.request(input, operations, requestConfig)),
-      TE.chain(response => {
+      TE.chainIOEitherK(response => {
```

Comment:
What's the difference between `chain` and `chainIOEitherK`?

Reply:
TE.chain requires the function to return a `TaskEither`; in this part we're internally returning an `Either` instead and so a transformation was required: `return TE.fromIOEither(r)`

fp-ts 2.4 introduced the ability of returning an Either from a TaskEither chain using the dedicated function, so we are not handing the transformation anymore.